acme/acme/client.py

```diff
@@ -286,19 +286,18 @@ def poll(self, authzr):
         # TODO: check and raise UnexpectedUpdate
         return updated_authzr, response
 
-    def request_issuance(self, csr, authzrs):
+    def request_issuance(self, csr, new_cert_uri):
         """Request issuance.
 
         :param csr: CSR
         :type csr: `OpenSSL.crypto.X509Req` wrapped in `.ComparableX509`
 
-        :param authzrs: `list` of `.AuthorizationResource`
+        :param new_cert_uri: new-cert URI
```

Comment:
Please leave a hint on how to get this URI: reference `AuhtorizationResource.new_cert_uri`